### DISCUSSION TITLE
57364A38-346B-7143-EEDB-2D2D868A1C85

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#147] [General]
# Abell 2

This add-on adds a realistic, fully 3D model of the planetary nebula Abell 2. Abell 2 is a very simple planetary nebula. Unfortunately its shape is not very well known; this is just a best guess based on available astrophotographs.

https://celestia.mobi/addon?item=57364A38-346B-7143-EEDB-2D2D868A1C85

**Comment by senormoonfv-sys** [comment]
abel
